cogs/TwitchAlert.py

```diff
@@ -693,8 +699,9 @@ async def requests_get(self, url, headers=None, params=None):
 
                 if response.status == 401:
```

Comment:
Is a 401 error given when we've reached the rate limit?
And do we not want to retry for different error codes and not just a 401? E.g. a 503 or 504?

Reply:
Performing it recursively like we do now but adding other error codes wouldn't result in an infinite loop. E.g. any request can have a max of 5 failed attempts before aborting no matter the error code returned.

Either way we don't handle other errors at all in this implementation.

Reply:
And for certain errors we can simply abort after 1 error code is recieved but we should definitely be throwing an error if there isn't a valid response.

Reply:
tbh all of this is working atm and is unrelated to this PR for fixing GitHub CI issues, please can they be made into a separate issue, or added to #105 so I can work on it after exams (I am already planning on remaking large parts of TwitchAlert)

Reply:
I think I'm going to move to using [pytwitchapi](https://pytwitchapi.readthedocs.io/en/latest/index.html) or a similar library so I don't have to maintain rate limiting etc.

Reply:
I have updated #105 to reflect these issues